draft-ietf-tls-rfc8446bis.md

```diff
@@ -5327,6 +5327,15 @@ Clients and Servers SHOULD NOT reuse a key share for multiple connections. Reuse
 of a key share allows passive observers to correlate different connections. Reuse
 of a client key share to the same server additionally allows the server to correlate different connections.
 
+It is RECOMMENDED that the labels for external identities be selected so that they
```

Comment:
FWIW, the actual protocol element is the "identity", which we describe as "a label for a key.  For instance...a label for a pre-shared key established externally".  We do use the phrasing "PSK identity" in the subsequent paragraph, so maybe "the identities used to identify (label) externally shared keys" if we want to increase consistency?